rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -2824,8 +2846,13 @@ bool game_list_frame::SearchMatchesApp(const QString& name, const QString& seria
 {
 	if (!m_search_text.isEmpty())
 	{
-		const QString search_text = m_search_text.toLower();
-		return m_titles.value(serial, name).toLower().contains(search_text) || serial.toLower().contains(search_text);
+		QString search_text = m_search_text.toLower();
+		QString title_name = m_titles.value(serial, name).toLower();
+
+		search_text.remove(reinterpret_cast<const char*>(u8"™"));
```

Comment:
How do I search for tm then?

Reply:
Why would you? that's not a character on your keyboard and the fact that you had remember which games has it on your list and where is unfathomable to me.

Reply:
It's unfathomable to me why you discriminate that one single special character in the first place

Reply:
![image](https://user-images.githubusercontent.com/18193363/233698372-88b8af14-f6cf-4d8e-ae1a-d8c43f8ec392.png)

Reply:
Made it be a fallback when no search results have been yielded,